gamemode/modules/base/sv_gamemode_functions.lua

```diff
@@ -919,14 +919,14 @@ function GM:PlayerLoadout(ply)
         end
     end
 
-    for _, v in pairs(self.Config.DefaultWeapons) do
+    for _, v in ipairs(self.Config.DefaultWeapons) do
```

Comment:
This is a config thing. It can be modified by users. There are some more of these.